src/fontra/views/editor/editor.js

```diff
@@ -1261,6 +1270,9 @@ export class EditorController {
   }
 
   canCut() {
+    if (this.sceneModel.isSelectedGlyphLocked() || this.fontController.readOnly) {
```

Comment:
Suggest to switch the two conditions: the readOnly check is cheaper, and if you do it first, it will shortcut the entire expression if readOnly is true, so `isSelectedGlyphLocked()` won't even be called. (Also the other occurrances.)

Reply:
Very good point, I have not thought of that. Thanks.